### ISSUE TITLE
Cannot build latest version 3.0.*

### ISSUE THREAD

**Issue by shenphen** [Lesha-spr/react-validation#120]
Below is output of building script. If i build it via npm, occurs same error. Issue points to versions 3.0.0 & 3.0.1
NPM v.5.4.2
Node v6.11.3
```
me@my-comp:~/X/Y/Z$ yarn add react-validation
yarn add v1.0.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.0.17: The platform "linux" is incompatible with this module.
info "fsevents@1.0.17" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "ajv-keywords@1.5.1" has incorrect peer dependency "ajv@>=4.10.0".
warning "marked-terminal@1.7.0" has incorrect peer dependency "marked@^0.3.3".
warning "babel-loader@7.1.1" has incorrect peer dependency "webpack@2 || 3".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "babel-eslint@^7.2.3".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "eslint@^4.1.1".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "eslint-plugin-flowtype@^2.34.1".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "eslint-plugin-import@^2.6.0".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "eslint-plugin-jsx-a11y@^5.1.1".
warning "eslint-config-react-app@2.0.1" has incorrect peer dependency "eslint-plugin-react@^7.1.0".
warning "extract-text-webpack-plugin@3.0.0" has incorrect peer dependency "webpack@^3.1.0".
warning "react-dom@16.0.0" has incorrect peer dependency "react@^16.0.0-beta.5".
warning "webpack-dev-server@2.8.2" has incorrect peer dependency "webpack@^2.2.0 || ^3.0.0".
warning "ajv-keywords@2.1.0" has incorrect peer dependency "ajv@>=5.0.0".
[4/4] Building fresh packages...
[2/2] ⠄ react-validation: ERR! This is probably not a problem with npm. There is likely additional logging output above.
[-/2] ⠄ waiting...
[-/2] ⠄ waiting...
[-/2] ⠄ waiting...
error /X/Y/Z/node_modules/react-validation: Command failed.
Exit code: 1
Command: npm run build
Arguments:
Directory: /X/Y/Z/node_modules/react-validation
Output:
> react-validation@3.0.1 build /X/Y/Z/node_modules/react-validation
> node scripts/build.js

Creating an optimized production build...
Failed to compile.

TypeError: Cannot read property 'request' of undefined

  - ExternalModuleFactoryPlugin.js:37 handleExternals
    [project-name]/[react-validation]/[html-webpack-plugin]/[webpack]/lib/Externa    lModuleFactoryPlugin.js:37:33

  - ExternalModuleFactoryPlugin.js:46 next
    [project-name]/[react-validation]/[html-webpack-plugin]/[webpack]/lib/Externa    lModuleFactoryPlugin.js:46:8

  - ExternalModuleFactoryPlugin.js:59 handleExternals
    [project-name]/[react-validation]/[html-webpack-plugin]/[webpack]/lib/Externa    lModuleFactoryPlugin.js:59:7

  - ExternalModuleFactoryPlugin.js:79 ExternalModuleFactoryPlugin.<anonymous>
    [project-name]/[react-validation]/[html-webpack-plugin]/[webpack]/lib/Externa    lModuleFactoryPlugin.js:79:5

  - NormalModuleFactory.js:234 applyPluginsAsyncWaterfall
    [project-name]/[react-validation]/[webpack]/lib/NormalModuleFactory.js:234:4

  - Tapable.js:268
    [project-name]/[react-validation]/[tapable]/lib/Tapable.js:268:11

  - IgnorePlugin.js:56 IgnorePlugin.checkIgnore
    [project-name]/[react-validation]/[webpack]/lib/IgnorePlugin.js:56:10

  - Tapable.js:272 NormalModuleFactory.applyPluginsAsyncWaterfall
    [project-name]/[react-validation]/[tapable]/lib/Tapable.js:272:13

  - NormalModuleFactory.js:218 NormalModuleFactory.create
    [project-name]/[react-validation]/[webpack]/lib/NormalModuleFactory.js:218:8

  - Compilation.js:382 Compilation._addModuleChain
    [project-name]/[react-validation]/[webpack]/lib/Compilation.js:382:17

  - Compilation.js:464 Compilation.addEntry
    [project-name]/[react-validation]/[webpack]/lib/Compilation.js:464:8

  - SingleEntryPlugin.js:22 SingleEntryPlugin.<anonymous>
    [project-name]/[react-validation]/[html-webpack-plugin]/[webpack]/lib/SingleE    ntryPlugin.js:22:15

  - Tapable.js:293 Compiler.applyPluginsParallel
    [project-name]/[react-validation]/[tapable]/lib/Tapable.js:293:14

  - Compiler.js:509 applyPluginsAsync.err
    [project-name]/[react-validation]/[webpack]/lib/Compiler.js:509:9

  - Tapable.js:195 Compiler.applyPluginsAsyncSeries
    [project-name]/[react-validation]/[tapable]/lib/Tapable.js:195:46

  - Compiler.js:502 Compiler.compile
    [project-name]/[react-validation]/[webpack]/lib/Compiler.js:502:8


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-validation@3.0.1 build: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-validation@3.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by Lesha-spr** [comment]
Hi @shenphen ,

Seems related to create-react-app. I'll investigate.

**Comment by Lesha-spr** [comment]
Hi again @shenphen ,

Just published 3.0.2 - removed unused loaders which may cause this issue. Could you please try to re-install?

Thanks

**Comment by shenphen** [comment]
Hello @Lesha-spr 

Now it's working perfectly ;)

Thank you a lot for your invaluable work.
